### ISSUE TITLE
Performance results

### ISSUE THREAD

**Issue by JamesChuanggg** [shmsw25/mcb-model-for-vqa#1]
Hi, Can you post statistic results of you implementation, like accuracy?

**Comment by csnemes2** [comment]
Hi,

I have tried this code with 'MCB_without_Attention' on 'mscoco' with the params which was in the repo.
I got poor results (Overall performance below 30%)
Far from the ones which were published in the paper ~ 60%

I played with learning rate, batch size, number of words used.  It was not successful.

Then I checked whether it learns on the training set at least. To speed up the process, I used only the first 300 images. The best i got so far was around 37%. Which hints that the code should work. I used the same vqaEval.py to minimize bugs. The whole training was very sensitive to the paramters.

Can you share what parameters you are using? What learning rate? What model parameters? How many runs? How many epoch? etc

It would be nice to have something which gives at least 50%.

**Comment by haibin894609937** [comment]
accuracy ?
